### PR TITLE
add BunkerWeb

### PR DESCRIPTION
**Pull request by fl0ppy-d1sk**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/bunkerweb.yml
Comment (Ki-er) [inline software/bunkerweb.yml:15]:
```suggestion
```

File: software/bunkerweb.yml
Comment (Ki-er) [inline software/bunkerweb.yml:4]:
```suggestion
description: "Next-gen Web Application Firewall (WAF) that will protect your web services."
```

File: software/bunkerweb.yml
Comment (Ki-er) [inline software/bunkerweb.yml:1]:
```suggestion
name: "Bunkerweb"
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes to be made.

**Review by ryangurn — CHANGES_REQUESTED** [review]
I think this will end up failing syntax-checks because of the double quotes around the description. You may want to remove the double quotes from this to prevent the "description does not end with a dot".

Example was my QST PR and the action that run initially.
* [Action](https://github.com/awesome-selfhosted/awesome-selfhosted-data/actions/runs/8810348676/job/24217675492)
* [PR](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674)

**Comment by fl0ppy-d1sk** [comment]
> I think this will end up failing syntax-checks because of the double quotes around the description. You may want to remove the double quotes from this to prevent the "description does not end with a dot".
> 
> Example below is my QST PR and the action that run initially.
> 
>     * [Action](https://github.com/awesome-selfhosted/awesome-selfhosted-data/actions/runs/8810348676/job/24217675492)
> 
>     * [PR](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/674)

Ok, thanks for the tip. Any thoughts @Ki-er ?

I've made the required changes btw.

**Comment by fl0ppy-d1sk** [comment]
Additional commit should fix the workflow.

**Comment by fl0ppy-d1sk** [comment]
Okay, `K8S` is in full uppercase, my bad.

**Review by nodiscc — APPROVED** [review]

File: software/bunkerweb.yml
Comment (nodiscc) [inline software/bunkerweb.yml:13]: To be fair this should probably go to https://github.com/awesome-foss/awesome-sysadmin?tab=readme-ov-file#web which lists Web servers and Reverse Proxies. The proxy section is for "traditional", outgoing proxies.

I'd like to do some reorganization (possibly move Web Servers/Reverse Proxies back to this list) before merging this.

Comment (fl0ppy-d1sk) [reply software/bunkerweb.yml:13]: Hello @nodiscc,

Thanks for the feedback. I can create a PR on awesome-sysadmin if you prefer ?

Comment (nodiscc) [reply software/bunkerweb.yml:13]: I will take care of the reorganization and merge this when I'm done. Assigning this to myself.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**
